nodeadm/internal/aws/ecr/ecr.go

```diff
@@ -49,8 +49,8 @@ func GetEKSRegistry(region string) (ECRRegistry, error) {
 	}
 	if fipsInstalled && fipsEnabled {
 		fipsRegistry := getRegistry(account, "ecr-fips", region, servicesDomain)
-		addresses, err := net.LookupHost(fipsRegistry)
-		if err == nil && len(addresses) > 0 {
+		resp, err := http.Get(fipsRegistry)
```

Comment:
`fipsRegistry` isn't a URL, right? It's just the registry's hostname. You would need to add a scheme for this to work

Reply:
You were right, it was failing everytime. My latest commit added the scheme, and I checked on AL2023 and RHEL. I was also able to hit both paths, with a VPC endpoint and without.